packages/ui/src/memberships/components/InviteMemberButton.tsx

```diff
@@ -14,21 +13,24 @@ interface InviteMemberButtonProps {
 }
 
 export const InviteMemberButton = ({ className, children, size }: InviteMemberButtonProps) => {
-  const [isOpen, toggleIsOpen] = useToggle()
+  const { showModal } = useModal()
   const { isTransactionPending } = useTransactionStatus()
 
+  const openModal = () => {
+    showModal({ modal: 'InviteMemberModal' })
+  }
```

Comment:
just a little shorter
```suggestion
  const openModal = () => showModal({ modal: 'InviteMemberModal' })
```